### ISSUE TITLE
Icon not available after a fresh install

### ISSUE THREAD

**Issue by linuxhorizon** [mjaschen/strava-helper-firefox#37]
Today I installed Strava Helper on Firefox (ver. 56.0 (64-bit) on Ubuntu 16.04 LTS). The plugin installation was without error but the Strava Helper icon is not shown and because of this I am unable to give "multi" kudos :). 
Of course, I restarted Firefox, and moreover, I done also several uninstall-install cycles. Nope, still not working.
I must mention also that on my laptop (same software configuration, I think) is working OK. 

Am I missing something?

**Comment by mjaschen** [comment]
Hi @rintux!

Sorry to hear that. I just installed the add-on from scratch and it works as expected. 

I there any chance that you've installed another add-on whicht is interfering?

**Comment by linuxhorizon** [comment]
No, not as far as I know. I had the same add-ons on my laptop and I had no problem.
Could you provide a link to try myself an installation from scratch?

**Comment by mjaschen** [comment]
Hi!

I did  the following steps:

1. Quit Firefox
2. move the Firefox profile directory to a temporary location
3. Start Firefox

Firefox starts with a new profile, i. e. without bookmarks, cookies, add-ons, etc.

Now I installed the Strava Helper add-on.

After verifying that the add-on is working as expected, I quit Firefox and moved the old profile back to its original location.

There's a tool called “Profile Manager” which makes the process of switching profiles a little bit easier, please take a look at [this page](http://kb.mozillazine.org/Profile_Manager#Accessing_the_Profile_Manager) for further details.

Hope that helps 🤞

**Comment by linuxhorizon** [comment]
Thanks a lot! 
I will try it tomorrow!
Keep in touch!
